src/PrestaShopBundle/Controller/Admin/Sell/CustomerService/OrderMessageController.php

```diff
@@ -244,6 +245,10 @@ private function getErrorMessages(): array
                 'The object cannot be loaded (or found)',
                 'Admin.Notifications.Error'
             ),
+            OrderMessageNameAlreadyUsedException::class => $this->trans(
```

Comment:
After **OrderMessageNameAlreadyUsedException** there's "::" IMO this is NOK, it should be ":" :slightly_smiling_face: 
I notice that with your PR, I have **Order Settings** instead of **Orders** and **Customer Settings** instead of **Customers**, I don't know if it's related but this is the only typo that I found :wink: